src/component/app.jsx

```diff
@@ -122,6 +119,7 @@ export default class App extends Component {
                             {createListItem('/applications', 'Applications', 'apps')}
                         </Navigation>
                     </Drawer>
+                    <ScrollContainer scrollKey="main">
```

Comment:
seems to work with same key for everything. Might want to use the current path (window.location.pathname)?